### ISSUE TITLE
[Feature request] label/goto, try/catch

### ISSUE THREAD

**Issue by Tomislav7** [SeleniumHQ/selenium-ide#191]
Hi!

It would be great if new Selenium IDE plugin for Firefox would support labels (ex. label x / goto x) and try/catch.
I've used this stuff with Flow Control and SelBlocks add-ons for Firefox but they are not supported any more.

Tnx!

**Comment by corevo** [comment]
It actually wouldn't be great if we supported labels and gotos.  
Labels and gotos are unstructured jumps, that create hard to follow code ([spaghetti code](https://en.wikipedia.org/wiki/Spaghetti_code)).  
Instead we are implementing a more structured version of [control flow](https://github.com/SeleniumHQ/selenium-ide/pull/140).  
I'm closing this for now, as it's both a duplicate and something we're working on.
